sys/linux/bpf.txt

```diff
@@ -97,11 +97,30 @@ bpf_prog {
 	prog_name		array[const[0, int8], BPF_OBJ_NAME_LEN]
 	prog_ifindex		ifindex[opt]
 	expected_attach_type	flags[bpf_attach_type, int32]
+	btf_fd			fd_btf
+	func_info_rec_size	len[func_info, int32]
+	func_info		ptr64[in, bpf_func_info]
+	func_info_cnt		int32
```

Comment:
It seems this should be either len or bytesize of func_info.

Reply:
func_info_cnt len[func_info, int32]